sites/main-site/src/pages/nft-utils.astro

```diff
@@ -0,0 +1,18 @@
+---
+import MarkdownTocLayout from '@layouts/MarkdownTocLayout.astro';
+
+const title = 'nft-utils';
+const subtitle = 'A collection of utilities for working with nf-test';
+
+const content = await fetch('https://raw.githubusercontent.com/nf-core/nft-utils/refs/heads/main/README.md');
+---
+
+<MarkdownTocLayout
+    title={title}
+    subtitle={subtitle}
+    headings={[]}
+    md_github_url="https://github.com/nf-core/nft-utils/edit/main/README.md"
+>
+    Lorem ipsum dolor sit amet consectetur, adipisicing elit. Iste, culpa! Optio itaque, nostrum earum sapiente pariatur
+    natus illum sit sunt quaerat similique. Labore aliquid voluptatibus sequi iste voluptatum! Itaque, voluptatum?
```

Comment:
Do I need to replace that with the readme or is the readme automatically rendered?

Reply:
I thought you wanted something more than the readme?

Reply:
I think if the README is rendered on the website that'll be good enough for me.
I just don't want to duplicate effort on writing docs

Reply:
@mashehu any comment/update?

Reply:
working on it, got hung up with the tools release sorry. is not sooo straight forward atm to add a random github repo, but there are ways.

Reply:
no worries, tools is more important, and I know you'll fix it, just wanted to check if it was not lost in limbo.
Thanks a lot